sdk/python/packages/flet/src/flet/map/__init__.py

```diff
@@ -1,33 +1 @@
-from flet.core.map.circle_layer import CircleLayer, CircleMarker
-from flet.core.map.map import (
-    Map,
-    MapEvent,
-    MapEventSource,
-    MapHoverEvent,
-    MapPointerDeviceType,
-    MapPointerEvent,
-    MapPositionChangeEvent,
-    MapTapEvent,
-)
-from flet.core.map.map_configuration import (
-    MapConfiguration,
-    MapInteractionConfiguration,
-    MapInteractiveFlag,
-    MapLatitudeLongitude,
-    MapLatitudeLongitudeBounds,
-    MapMultiFingerGesture,
-)
-from flet.core.map.marker_layer import Marker, MarkerLayer
-from flet.core.map.polygon_layer import PolygonLayer, PolygonMarker
-from flet.core.map.polyline_layer import (
-    DashedStrokePattern,
-    DottedStrokePattern,
-    PatternFit,
-    PolylineLayer,
-    PolylineMarker,
-    SolidStrokePattern,
-)
-from flet.core.map.rich_attribution import RichAttribution
-from flet.core.map.simple_attribution import SimpleAttribution
-from flet.core.map.text_source_attribution import TextSourceAttribution
-from flet.core.map.tile_layer import MapTileLayerEvictErrorTileStrategy, TileLayer
+
```

Comment:
Not sure about this too. How someone supposed to import Map controls?
We are going to move `flet_*` projects from the core anyway, so devs could get used to do `import flet.map` already.

Python packages will be called `flet-*` and have `flet.<something>` inside, so for example installing `flet-webview` will make available `flet.webview` module in your Python project.